### PR TITLE
Lorentzian

### PR DESCRIPTION
**Pull request by lcwheeler**
Add a LorentzianSpectrum class as per Zach's recipe for new Spectrum classes.

### CODE REVIEW

**Comment by Zsailer** [comment]
Nice! What do you think about the new base class?

File: ambigauss/__init__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 from .curves import gaussian, lorentzian
-from .spectra import GaussianSpectrum
+from .spectra import GaussianSpectrum, Lorentzian
```
Comment (Zsailer) [inline ambigauss/__init__.py]: bug here--should be `LorentzianSpectrum`

Comment (lcwheeler) [reply ambigauss/__init__.py]: Oh wow, I messed up the easiest thing ever, haha.

**Comment by lcwheeler** [comment]
The new base class is awesome! Much more flexible than what we had before.

File: ambigauss/__init__.py (hunk 1 above)
Comment (lcwheeler) [reply ambigauss/__init__.py]: Fixed :+1:

**Pull request merged**